examples/init-wait/src/wasm.rs

```diff
@@ -29,15 +29,6 @@ static mut STATE: State = State::NotInited;
 static mut INIT_MESSAGE: MessageId = MessageId::new([0; 32]);
 static mut TEST_DYNAMIC_MEMORY: BTreeMap<u32, ()> = BTreeMap::new();
 
-#[no_mangle]
```

Comment:
grinding affected lines count?)

Reply:
I've broken my brain because of non-standard function position